f2/src/forms/WhoAreYouReportForForm.js

```diff
@@ -11,6 +11,19 @@ import { P } from '../components/formik/paragraph'
 import { NextCancelButtons } from '../components/formik/button'
 import { formDefaults } from './defaultValues'
 import { WhoAreYouReportForFormSchema } from './WhoAreYouReportForFormSchema'
+import { ErrorSummary } from '../components/formik/alert'
+
+const createErrorSummary = (errors) => {
+  const errorSummary = {}
+  if (errors.whoYouReportFor) {
+    errorSummary['whoYouReportFor'] = {
+      label: <Trans id="whoAreYouReportForPage.title" />,
```

Comment:
The link in the ErrorSummary component will scroll to the element with ID **whoYouReportFor**. Right now there is no element with this ID so it simply scrolls to the top. Please update the error message to contain a unique ID so the link will work properly.

Reply:
Thank you for teaching, fixed it.